tap4j/src/main/java/org/tap4j/parser/Tap13Parser.java

```diff
@@ -239,9 +263,12 @@ public TestSet parseTapStream(Readable tapStream) {
     /**
      * Parse a TAP line.
      *
-     * @param tapLine TAP line
+     * @param tapLineOrig TAP line
      */
-    public void parseLine(String tapLine) {
+    public void parseLine(String tapLineOrig) {
+
+        // filter out cursor related control sequences ESC[25?l and ESC[25?h
+        String tapLine = tapLineOrig.replaceAll("\u001B\\[\\?25[lh]", "");
```

Comment:
Could you explain why we need this, and why just these two escape sequences, please?

Reply:
The two sequences are to show/hide cursor. I remember only that i needed the change in order to be able to process some TAP files, generated within our NPM (https://www.npmjs.com/) mass testing infrastructure. Unfortunately i did not keep the details on the concrete NPM package or test framework producing such characters inside the TAP stream. It was rather a "hot fix" back then and at the moment i am sorry that i did not spend more time on it and did not work it out better. However, i believe this minor change could be useful for others who might otherwise also experience issues with these control characters. If there is a need, the control character replacing mechanism could be further improved.

Reply:
I apologize that i do not have a better answer at the moment. Please let me know if it is still o.k. to merge.